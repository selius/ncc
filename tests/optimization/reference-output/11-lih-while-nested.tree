main:
{ }
|- =
|  |- i
|  `- 0
|- =
|  |- j
|  `- 0
|- =
|  |- d
|  `- 4
|- =
|  |- a
|  `- *
|     |- 5
|     `- d
|- while
|  |- <
|  |  |- i
|  |  `- 5
|  `- { }
|     |- =
|     |  |- j
|     |  `- 0
|     |- =
|     |  |- b
|     |  `- *
|     |     |- i
|     |     `- 5
|     |- while
|     |  |- <
|     |  |  |- j
|     |  |  `- 5
|     |  `- { }
|     |     |- =
|     |     |  |- c
|     |     |  `- *
|     |     |     |- i
|     |     |     `- j
|     |     `- ++(postfix)
|     |        `- j
|     `- ++(postfix)
|        `- i
|- __print_int()
|  `- a
|- __print_int()
|  `- b
|- __print_int()
|  `- c
`- return
   `- 0
