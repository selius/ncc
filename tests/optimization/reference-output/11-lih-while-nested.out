20
20
16
